pyquil/api/compiler.py

```diff
@@ -161,3 +166,69 @@ def get_job_fn():
         return wait_for_job(get_job_fn,
                             ping_time if ping_time else self.ping_time,
                             status_time if status_time else self.status_time)
+
+    def _pipo_payload(self, clifford, pauli):
```

Comment:
why is this pipo?

Reply:
pauli-in-pauli-out - I can change it to something more verbose/clear, if you think that would be better. (I think it probably would be.)